src/components/Text/Annotated/AnnotatedText.tsx

```diff
@@ -2,50 +2,105 @@ import { BroccoliTextGeneric } from "../../../model/Broccoli.ts";
-import { createLineOffsets } from "./utils/createLineOffsets.ts";
-import { createLineSearchOffsets } from "./utils/createLineSearchOffsets.ts";
-import { getAnnotationsByTypes } from "./utils/getAnnotationsByTypes.ts";
+import { createSearchOffsets } from "./utils/createSearchOffsets.ts";
 import { useDetailUrlParams } from "./utils/useDetailUrlParams.tsx";
 import "./annotated.css";
+import {
+  projectConfigSelector,
+  useProjectStore,
+} from "../../../stores/project.ts";
+import _ from "lodash";
+import {
+  AnnoRepoAnnotation,
+  isLogicalTextAnchorTarget,
+} from "../../../model/AnnoRepoAnnotation.ts";
+import {
+  createFootnoteMarkLineOffsets,
+  createNestedLineOffsets,
+  createPageMarkLineOffsets,
+} from "./utils/createLineOffsets.ts";
+import { LineOffsets } from "./AnnotationModel.ts";
 
 type TextHighlightingProps = {
   text: BroccoliTextGeneric;
   showDetail: boolean;
 };
 
+export const DUMMY_ID = "urn:suriano:file:1697716";
+
 /**
- * Definitions:
- * - Logical text: 'doorlopende' text, not split by line breaks
- * - Line: piece of annotated text as received from broccoli, a broccoli 'line' can also contain a logical text
- * - Annotation offset: character index at which an annotation starts or stops
- * - Character index: start index marks first character to include, stop index marks first character to exclude
- * - Line segment: piece of line uninterrupted by annotation offsets
- * - Annotation segment: piece of an annotation uninterrupted by the offsets of other overlapping/nested annotations
- * - Annotation group: all annotations that are connected to each other by other overlapping/nested annotations
- * - Annotation depth: the number of levels that an annotation is nested in parent annotations or with overlapping annotations
+ * Annotation definitions
+ * - Depth: the number of levels that an annotation is nested in parent annotations or overlapping annotations
  *   (when two annotations overlap, the second annotation has a depth of 2)
+ * - Group: all annotations or segments that are connected to each other by other overlapping/nested annotations
+ * - Line: piece of annotated text as received from broccoli, a broccoli 'line' can also contain a logical text
+ * - Marker: annotation of zero length, useful for marking locations of footnotes, page endings, etc.
+ * - Offset: start or end character index of an annotation in a line
+ *   - start index marks first character to include
+ *   - stop index marks first character to exclude
+ * - Segment: piece of line or annotation uninterrupted by annotation offsets
  */
 export const AnnotatedText = (props: TextHighlightingProps) => {
+  const { footnoteMarkerAnnotationTypes, pageMarkerAnnotationTypes } =
+    useProjectStore(projectConfigSelector);
   const annotations = useAnnotationStore().annotations;
 
   const { tier2, highlight } = useDetailUrlParams();
   const searchTerms = highlight;
+  // const isDummy = DUMMY_ID === tier2;
```

Comment:
50 en 51 kunnen weg, lijkt mij?

Reply:
Fixed in [8988b27](https://github.com/knaw-huc/textannoviz/pull/154/commits/8988b2734d0fa14a4d9f8206dc51dc3b236b484f)